statement_exercise.py

```diff
@@ -0,0 +1,45 @@
+def atm_machine():
+    current_balance = 10000
+    min_withdrawal = 500
+    max_withdrawal = 5000
+    move_on = True 
+  
+    while move_on:
+        print("Welcome to the ATM machine")
+
+        choice = input("Do you want to withdraw money? (yes/no): ")
```

Comment:
also remeber to add a condition when the user puts a choice that is not yes or no